CHANGELOG.md

```diff
@@ -1,7 +1,24 @@
-# dbt_qualtrics_source version.version
+# dbt_qualtrics_source v0.4.0
 
-## Documentation
-- Corrected references to connectors and connections in the README. ([#14](https://github.com/fivetran/dbt_qualtrics_source/pull/14))
+[PR #16](https://github.com/fivetran/dbt_qualtrics_source/pull/16) and [PR #17](https://github.com/fivetran/dbt_qualtrics_source/pull/17) include the following updates:
+
+## Schema & Data Updates
+**1 new column introduced and 1 schema change**
```

Comment:
Should we call this out as a breaking change?

Reply:
Since no jobs will actually break as a result of this new column, I opted to call it out as a schema change instead. For existing users there's no need to take action unless they want to leverage the new field.

Reply:
I updated the wording of this section to be more clear about the schema change and how it's not actually a breaking change.